bin/etcd-manage

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -eo pipefail
+
+preStart() {
+  # pull ip address for current machine and set in environment
+  export ETCD_IP=$(hostname -i)
+  _log "Started etcd at ${ETCD_IP}"
+}
+
+# TODO: Need to stop etcd properly
```

Comment:
> having removed ContainerPilot

I wanted to ask you about that. It's ok to remove it, but if you run into enough situations like this, you might prefer to bring it back (including the Consul dependency). Basically, don't remove it for purity's sake if it has practical use.